mlir/lib/Dialect/LLVMIR/IR/NVVMDialect.cpp

```diff
@@ -880,7 +880,7 @@ LogicalResult NVVM::WgmmaMmaAsyncOp::verify() {
   // Check transpose (only available for f16/bf16)
```

Comment:
Can we add a comment here that mentions the required layouts. We can use ehat PTX guideline says

Reply:
ok, added some description according to PTX guideline